### ISSUE TITLE
Drop download.opensuse.org ?

### ISSUE THREAD

**Issue by davidkarlsen** [myoung34/docker-github-actions-runner#266]
Is the download.opensuse.org apt repo really needed? It fails now due to:

```shell
Reading package lists...
W: GPG error: https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_20.04  InRelease: The following signatures were invalid: EXPKEYSIG 4D6439037[50](https://github.com/tietoevryfs/kyc-case-service/actions/runs/3630053351/jobs/6123487425#step:7:51)60AA4 devel:kubic OBS Project <devel:kubic@build.opensuse.org>
E: The repository 'https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_20.04  InRelease' is not signed.
```

**Comment by Uzlopak** [comment]
see #256

**Comment by nmalaguti** [comment]
This should be fixed by https://github.com/myoung34/docker-github-actions-runner/pull/265

The opensuse repo is used to add podman to 20.04. I think we should definitely consider whether that should be in the base image once #259 is complete.

**Comment by Uzlopak** [comment]
Ah lol, wanted to link 265 but linked 256. My bad.

**Comment by myoung34** [comment]
Yup, fixed with #265
